cuda_bindings/docs/source/release/13.X.Y-notes.rst

```diff
@@ -12,7 +12,16 @@ Released on TBD
 Highlights
 ----------
 
+* Migrated wheel dependencies from individual NVIDIA packages to the ``cuda-toolkit`` metapackage for improved dependency resolution and version constraints.
 * Automatic CUDA library path detection based on ``CUDA_HOME``, eliminating the need to manually set ``LIBRARY_PATH`` environment variables for installation.
+* Removed NVCC compiler from ``[all]`` optional dependencies as it was previously included only to access NVVM library, which now has its own dedicated wheel.
```

Comment:
No need to highlight this. The text is good, though. Let merge it with what you added below.